tools/postinstall/apply-patches.js

```diff
@@ -98,6 +98,10 @@ searchAndReplace(
 // Workaround for: https://github.com/bazelbuild/rules_nodejs/issues/1208.
 applyPatch(path.join(__dirname, './manifest_externs_hermeticity.patch'));
 
+// Pre-req for. https://github.com/angular/components/pull/18960. Can be removed
```

Comment:
This patch is a prereq for itself?  I am not quite sure I follow what you are expressing here.

Reply:
It's basically a way to avoid the cyclic dependency. e.g. `angular/angular` updates to `rules_nodejs#1.5.0` with patches for `@angular/bazel`. Then `components-repo-unit-tests` fails because it still uses an older `rules_nodejs` version and `@angular/bazel` is no longer compatible due to the new changes.

To fix this, the components repo needs to update to `rules_nodejs#1.5.0`, but at the same time that update needs the compatibility changes for `@angular/bazel`. Hence the patch.

Reply:
You are both correct :) That was the incorrect link pasted there. Copy pasta. Fixed the link to point to the Angular PR this is a pre-req for.

Reply:
Ops. Looks like I misread that the URL pointed to _this_ PR 😅